docs/docs/CHANGELOG.md

```diff
@@ -1,5 +1,508 @@
+- rm cli code [\#2824](https://github.com/pomerium/pomerium/pull/2824) (@wasaga)
+- fix typo in docs [\#2819](https://github.com/pomerium/pomerium/pull/2819) (@wasaga)
+
+## [v0.15.8](https://github.com/pomerium/pomerium/tree/v0.15.8) (2021-12-17)
+
+[Full Changelog](https://github.com/pomerium/pomerium/compare/v0.15.7...v0.15.8)
+
+## Fixed
+
+- authorize: fix nginx infinite redirect [\#2812](https://github.com/pomerium/pomerium/pull/2812) (@calebdoxsey)
+
+## Documentation
+
+- DOCS: Fix indentation in API doc [\#2799](https://github.com/pomerium/pomerium/pull/2799) (@backport-actions-token[bot])
+- Docs: Update Kubernetes Dashboard Guide [\#2795](https://github.com/pomerium/pomerium/pull/2795) (@backport-actions-token[bot])
+- Docs: Update Securing Kubernetes Guide [\#2792](https://github.com/pomerium/pomerium/pull/2792) (@backport-actions-token[bot])
+- docs: update pomerium-cli location [\#2790](https://github.com/pomerium/pomerium/pull/2790) (@travisgroth)
+- Document Pomerium Policy Language [\#2789](https://github.com/pomerium/pomerium/pull/2789) (@backport-actions-token[bot])
+- Docs: Update JWT Verification Guide [\#2787](https://github.com/pomerium/pomerium/pull/2787) (@backport-actions-token[bot])
+
+## Dependency
+
+- deps: pin release to latest go version [\#2827](https://github.com/pomerium/pomerium/pull/2827) (@travisgroth)
+
+## [v0.15.7](https://github.com/pomerium/pomerium/tree/v0.15.7) (2021-11-15)
+
+[Full Changelog](https://github.com/pomerium/pomerium/compare/v0.15.6...v0.15.7)
+
+## Fixed
+
+- autocert: remove log [\#2750](https://github.com/pomerium/pomerium/pull/2750) (@backport-actions-token[bot])
+
+## Security
+
+- identity: fix user refresh [\#2725](https://github.com/pomerium/pomerium/pull/2725) (@backport-actions-token[bot])
+
+## Documentation
+
+- Docs: Add Grafana Integration Guide [\#2762](https://github.com/pomerium/pomerium/pull/2762) (@backport-actions-token[bot])
+- Docs: Add spdy annotation [\#2751](https://github.com/pomerium/pomerium/pull/2751) (@backport-actions-token[bot])
+- Docs: Add spdy annotation [\#2747](https://github.com/pomerium/pomerium/pull/2747) (@alexfornuto)
+- Docs: Ingress Controller [\#2745](https://github.com/pomerium/pomerium/pull/2745) (@backport-actions-token[bot])
+- Docs: Update Traefik Example Headers [\#2741](https://github.com/pomerium/pomerium/pull/2741) (@backport-actions-token[bot])
+- Docs: Update Community Page [\#2731](https://github.com/pomerium/pomerium/pull/2731) (@backport-actions-token[bot])
+- Minor fix in routes documentation [\#2721](https://github.com/pomerium/pomerium/pull/2721) (@backport-actions-token[bot])
+- Docs: Reference gRPC API Docs [\#2720](https://github.com/pomerium/pomerium/pull/2720) (@backport-actions-token[bot])
+- Update architecture.md [\#2707](https://github.com/pomerium/pomerium/pull/2707) (@backport-actions-token[bot])
+
+## [v0.15.6](https://github.com/pomerium/pomerium/tree/v0.15.6) (2021-11-04)
```

Comment:
This entry needs to be manually consolidated with v0.16.0 when we're ready to release..  The misplaced tag confuses the changelog generator.